cbor/examples/cbor/main/idf_component.yml

```diff
@@ -0,0 +1,6 @@
+## IDF Component Manager Manifest File
+version: "1.0.0"
+description: CBOR Example
+dependencies:
+  espressif/cbor:
+    override_path: '../../../'
```

Comment:
`override_path` states that the example should use the local component instead of getting it from the component manager. I don't think not giving the version would make any difference.

Reply:
> override_path states that the example should use the local component instead of getting it from the component manager

Only when you cloned the example directly from the component repository. When you set up the example using component manager cli then it shall remove the `override_path` and hence version becomes important.

Reply:
Oh okay. Got it.